tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -754,26 +754,41 @@ def args_dict_to_list(cli_parser, **kwargs):
 
 
 def pack_params_to_args_namespace(**kwargs):
+    # This method converts dictionary of params from convert_model() to argparse.Namespace object,
+    # checks that parameters are known and creates list of non-default params for serialization in IR
+
+    # create MO cli parser
     fe_manager = FrontEndManager()
     cli_parser = get_all_cli_parser(fe_manager)
+
+    # parse params from convert_model(), the result is argparse.Namespace object
     argv = cli_parser.parse_args(args_dict_to_list(cli_parser, **kwargs))
 
+    # get list of all available params for convert_model()
     all_params = {}
     for key, value in mo_convert_params.items():
         all_params.update(value)
```

Comment:
Here I didn't understand how we update the dictionary with just `ParamDescription` object. Can you please clarify?

Reply:
There was a change in format of `mo_convert_params` to make it contain groups of parameters like "tf", "kaldi" and so on. To store such information I made `mo_convert_params `a dictionary where key is category ("tf", "onnx"...) and value is dictionary of parameters (key - name, value - `ParamDescription`). 
So in this cycle I get parameters from all categories.